web/profiles/joinup/src/Form/SolutionsByLicenceForm.php

```diff
@@ -159,28 +153,31 @@ protected function getLicenceOptions(): array {
    */
   protected function getCountQuery(string $licence_id = NULL): string {
     $query = <<<QUERY
-SELECT COUNT(*) as ?total
```

Comment:
Please note, that there was an update in the query to ensure that if a solution has more than one distribution referencing the same licence, there will be only one result shown in the table.